xkcd/pkg/index/index.go

```diff
@@ -0,0 +1,158 @@
+package index
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/vespian/go-exercises/xkcd/pkg/pbuff"
+	"github.com/vespian/go-exercises/xkcd/pkg/types"
+	"github.com/vespian/go-exercises/xkcd/pkg/web"
+)
+
+func serialize(a types.AllStories, t types.OndiskSerialization,
+) (
```

Comment:
This newline is unlikely to be canonical style - I suggest running go fmt on this excersize's files?